packages/template-retail-react-app/app/components/experience/carousel/index.jsx

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React, {Fragment, useCallback, useRef} from 'react'
+import PropTypes from 'prop-types'
+import {AspectRatio, Box, Heading, IconButton, Stack, useBreakpointValue} from '@chakra-ui/react'
+import {Component} from 'commerce-sdk-react-preview/components'
```

Comment:
A reminder that I understood we mentioned that we want to import these components directly in the template for now until we release the new commerce-sdk package.

Reply:
From the sounds of it, we'll be copying over those components from the library into the template, we'll have to update this import.